packages/backend/migrations/20210615140530_add_account_id_to_users_table.js

```diff
@@ -0,0 +1,11 @@
+exports.up = function (knex) {
```

Comment:
Migrations are great for progressively upgrading/downgrading a live database, but since there are no live Rafiki instances yet it would be simpler to just add the new fields directly to the original table setup migration.